common/meta/raft-store/src/state_machine/sm.rs

```diff
@@ -559,6 +576,241 @@ impl StateMachine {
         Ok(Change::new(prev, result).into())
     }
 
+    fn return_value_condition_result(
+        &self,
+        expected: i32,
+        target_value: &Vec<u8>,
+        value: SeqV,
```

Comment:
This method does not consume `value`, using a reference would be better: `value: &SeqV`